tests/test_models.py

```diff
@@ -958,3 +958,10 @@ def test_datamodel_save_filename(tmp_path):
 
     with datamodels.open(filename) as new_ramp:
         assert new_ramp.meta.filename == filename.name
+
+
+@pytest.mark.parametrize("model_class", [datamodels.RampModel, datamodels.ScienceRawModel, datamodels.TvacModel])
+def test_rampmodel_from_science_raw(model_class):
+    """Test creation of RampModel from raw science/tvac"""
+    model = utils.mk_datamodel(model_class)
+    ramp = datamodels.RampModel.from_science_raw(model)
```

Comment:
Maybe add a line where you set a value and confirm the set value is properly propagated?